programs/keeper-client/Commands.cpp

```diff
@@ -441,7 +442,7 @@ void ReconfigCommand::execute(const DB::ASTKeeperQuery * query, DB::KeeperClient
             new_members = query->args[1].safeGet<String>();
             break;
         default:
-            UNREACHABLE();
+            throw Exception(ErrorCodes::UNEXPECTED_ZOOKEEPER_ERROR, "Unexpected operation: {}", operation);
```

Comment:
What about using `LOGICAL_ERROR`? Reaching a previously unreachable part in the code for me is definitely a logical error. We track logical errors in cloud, just as crashes (that would be the result of reaching `UNREACHABLE()`), but not necessarily `UNEXPECTED_ZOOKEEPER_ERROR`.

Reply:
Fair enough, thanks. I'll change to `LOGICAL_ERROR`. (need to merge from upstream anyways as CI is broken again)